lib/serverengine/supervisor.rb

```diff
@@ -194,20 +239,29 @@ def try_join
     end
 
     def start_server
-      s = create_server(logger)
-      @last_start_time = Time.now
+      unless ServerEngine.windows?
```

Comment:
Normally, I don't prefer `unless ... else ... end`.
But in this case, code for unix-like systems should be prior. So it's better to add code comment at the line of `else` ("ServerEngine.windows?" or something like that).